formstack-baton-requests/src/main/scala/com/gu/identity/formstackbatonrequests/aws/S3Client.scala

```diff
@@ -0,0 +1,67 @@
+package com.gu.identity.formstackbatonrequests.aws
+
+import com.amazonaws.services.s3.AmazonS3Client
+import com.amazonaws.services.s3.model.S3ObjectSummary
+import com.gu.identity.formstackbatonrequests.SarLambdaConfig
+import com.typesafe.scalalogging.LazyLogging
+
+import scala.collection.JavaConverters._
+import scala.util.Try
+
+sealed trait S3StatusResponse
+case class S3CompletedPathFound(resultLocations: List[String]) extends S3StatusResponse
+case class S3FailedPathFound() extends S3StatusResponse
+case class S3NoResultsFound() extends S3StatusResponse
+
+
+trait S3Client {
+  def checkForResults(initiationId: String, config: SarLambdaConfig): Either[Throwable, S3StatusResponse]
+}
+
+object S3 extends S3Client with LazyLogging {
```

Comment:
This object and trait does not contain changes from https://github.com/guardian/identity-processes/pull/52/files, it's just showing up as a change because I've moved it into its own file